modules/system/classes/MediaLibrary.php

```diff
@@ -636,9 +636,20 @@ protected function initLibraryItem($item, $itemType)
             $size = $this->getFolderItemCount($item['path']);
         }
 
+        /*
+         * set image dimensions,
+         * folders dimensions is always NULL.
+         */
+        if ($itemType === MediaLibraryItem::TYPE_FILE) {
+            $imageDimensions = getimagesize($this->getStorageDisk()->path($item['path']));
+            $dimensions = "$imageDimensions[0] x $imageDimensions[1]";
+        } else {
+            $dimensions = null;
+        }
```

Comment:
`TYPE_FILE` would also include videos, audio, etc.

It this could be limited by only applying this logic to `$item->getFileType() === MediaLibraryItem::FILE_TYPE_IMAGE`, however that will not be available until after the `MediaLibraryItem` has been inited.